### DISCUSSION TITLE
Intent for CI?

### DISCUSSION THREAD

**Discussion by mpilgrem** [reanimate/reanimate#238] [Q&A]
May I ask what is the intent for the CI for the repository? I get the impression that the scripts may be a little out of date.

I see GitHub Actions/haskell.yml refers to ubuntu and GHC 8.8.2 using cabal; and the Azure templates refer to ubuntu, macOS and Windows and GHC 8.10.7, GHC 8.8.3 (LTS 15.16, rather than LTS 16.11) and GHC 8.6.5 using stack.

**Comment by lemmih** [comment]
The purpose of both the GitHub Actions and the Azure Pipeline is the same: making sure reanimate works with the given stack-lts-*.yaml files. Some versions don't work on some platforms, though. Feel free to make any changes you see fit.
